data/BejadiRajeshReddy18.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "RajeshReddy",
+  "displayStatsPublic":true
```

Comment:
As of latest update, we no longer need `displayStatsPublic` key. Please remove this one!

```suggestion
```